exampleCourse/questions/demo/autograder/python/orderBlocksAddNumpy/tests/test.py

```diff
@@ -0,0 +1,22 @@
+from pl_helpers import name, points, not_repeated
```

Comment:
Is there a reason there is an empty file just before this one? 

[sorry for comment in the wrong place, couldn't figure out how to comment on the empty file because there are no lines to comment on]

Reply:
I agree it seems unnecessary, but I'm going to resolve this now to get this merged.